src/app/games/diaballik/DiaballikTutorial.ts

```diff
@@ -14,7 +14,7 @@ export class DiaballikTutorial extends Tutorial {
     public tutorial: TutorialStep[] = [
         TutorialStep.informational(
             $localize`Goal of the game`,
-            $localize`The goal of Diaballik is to bring your ball, currently being held by your center piece as represented by two circles, into the home line of the opponent.`,
+            $localize`The goal of Diaballik is to bring your ball, currently being held by your center piece as represented by the small circle, into the home line of the opponent.`,
             DiaballikState.getInitialState(),
```

Comment:
The goal of Diaballik is to bring your ball, represented by the small circle, into the home line of the opponent.
The ball is currently being held by your center piece.

Voir à check si "home line" est bien clair

Reply:
"the home line (i.e., its starting position)" pour être sûr ?